integration-cli/docker_utils.go

```diff
@@ -1564,6 +1564,7 @@ func setupRegistry(c *check.C) *testRegistryV2 {
 		if err = reg.Ping(); err == nil {
 			break
 		}
+		c.Log(err)
```

Comment:
@runcom This is weird though, it should have printed out the error if the assert failed.. :sweat: